### PR TITLE
Add Command Summary

### LINKED ISSUE #281: UG - Command Summary

### PR DESCRIPTION
**Pull request by juliusgambe**
Close #281

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -625,3 +625,45 @@ You can delete all your customer and delivery data by calling this command._
...
+| Login    | `login --user USERNAME --password PASSWORD`                                                                                  | `login --user gabriel --password gabrielIsGreat`                                                                            |
+| Update   | `update [--user USERNAME] [--password PASSWORD --confirmPass CONFIRM_PASSWORD] [--secretQn SECRET_QUESTION --answer ANSWER]` | `login --user gabrielV2 --password gabrielIsBest --confirmPass gabrielIsBest --secretQn Favourite Pet --answer BoBo`        |
+| Recover  | `recover account [--answer ANSWER --password NEW_PASSWORD --confirmPass CONFIRM_PASSWORD]`                                   | `recover account --answer Koko --password newPassword123 --confirmPass newPassword123`                                      |
+| Logout   | `logout`                                                                                                                     | `logout`                                                                                                                    |
+
```
Comment (B-enguin) [inline docs/UserGuide.md]: Would this be missing Account Deletion

File: docs/UserGuide.md [hunk 2]
```diff
@@ -625,3 +625,45 @@ You can delete all your customer and delivery data by calling this command._
...
+|---------|------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| Add     | `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`           | `customer add --name Gabriel --phone 87654321 --email` |
+| Find    | `customer find KEYWORD [MORE_KEYWORDS]`                                                  | `customer find Ibrahim Yu`                                                                         |
+| List    | `customer list`                                                                          | `customer list`                                                                                    |
+| Edit    | `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]` | `customer edit 1001 --name Gabriel -–phone 1234 5678 --email` |
+| Delete  | `customer delete CUSTOMER_ID`                                                            | `customer delete 1001`                                                                             |
+
```
Comment (B-enguin) [inline docs/UserGuide.md]: Would this be missing Customer View?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -625,3 +625,45 @@ You can delete all your customer and delivery data by calling this command._
...
+| View    | `delivery view DELIVERY_ID`                                                               | `delivery view 1001`                                                                               |
+| Status  | `delivery status STATUS CUSTOMER_ID`                                                      | `delivery status completed 2`                                                                      |
+| Edit    | `delivery edit DELIVERY_ID [--name DELIVERY_NAME] [--customer CUSTOMER_ID] [--date DELIVERY_DATE] [--status STATUS] [--note NOTE]` | `delivery edit 1001 --name Chocolate Cake --customer 2 --date 2024-12-12 --status CANCELLED --note Customer changed his mind.` |
+| Note    | `delivery note DELIVERY_ID --note NOTE`                                                   | `delivery note 1 --note By FedEx`                                                                  |
+| Delete  | `delivery delete DELIVERY_ID`                                                             | `delivery delete 1001`                                                                             |
+
```
Comment (B-enguin) [inline docs/UserGuide.md]: Would this be missing Delivery Find?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -625,3 +625,45 @@ You can delete all your customer and delivery data by calling this command._
...
+| Login    | `login --user USERNAME --password PASSWORD`                                                                                  | `login --user gabriel --password gabrielIsGreat`                                                                            |
+| Update   | `update [--user USERNAME] [--password PASSWORD --confirmPass CONFIRM_PASSWORD] [--secretQn SECRET_QUESTION --answer ANSWER]` | `login --user gabrielV2 --password gabrielIsBest --confirmPass gabrielIsBest --secretQn Favourite Pet --answer BoBo`        |
+| Recover  | `recover account [--answer ANSWER --password NEW_PASSWORD --confirmPass CONFIRM_PASSWORD]`                                   | `recover account --answer Koko --password newPassword123 --confirmPass newPassword123`                                      |
+| Logout   | `logout`                                                                                                                     | `logout`                                                                                                                    |
+
+#### Customer
```
Comment (B-enguin) [inline docs/UserGuide.md:746]: Should we arrange Customer and Delivery to be listed in the same order, e.g. Add, Find, List

**Review by B-enguin — COMMENTED** [review]
Some General Comments

File: docs/UserGuide.md [hunk 5]
```diff
@@ -636,3 +637,47 @@ You can delete all your customer and delivery data by calling this command._
...
+| Add     | `delivery add DELIVERY_NAME --customer CUSTOMER_ID --date DELIVERY_DATE`                  | `delivery add furniture --customer 5 --date 2023-12-03`                                           |
+| Find    | `delivery find KEYWORD [MORE_KEYWORDS]`                                                  | `delivery find Ibrahim Yu`                                                                         |
+| List    | `delivery list [--status STATUS] [--customer CUSTOMER_ID] [--date DELIVERY_DATE] [--sort SORT]` | `delivery list --status created --customer 1 --date 2023-12-12 --sort desc`                        |
+| View    | `delivery view DELIVERY_ID`                                                               | `delivery view 1001`                                                                               |
+| Status  | `delivery status STATUS CUSTOMER_ID`                                                      | `delivery status completed 2`                                                                      |
```
Comment (B-enguin) [inline docs/UserGuide.md]: Should this be `DELIVERY_ID` instead

Comment (B-enguin) [reply docs/UserGuide.md]: Also status takes ID before the Status

**Review by B-enguin — COMMENTED** [review]
A few more comments

File: docs/UserGuide.md [hunk 6]
```diff
@@ -636,3 +637,47 @@ You can delete all your customer and delivery data by calling this command._
...
+| Edit    | `delivery edit DELIVERY_ID [--name DELIVERY_NAME] [--customer CUSTOMER_ID] [--date DELIVERY_DATE] [--status STATUS] [--note NOTE]` | `delivery edit 1001 --name Chocolate Cake --customer 2 --date 2024-12-12 --status CANCELLED --note Customer changed his mind.` |
```
Comment (B-enguin) [inline docs/UserGuide.md]: Note does not allow symbols after #324

File: docs/UserGuide.md [hunk 7]
```diff
@@ -636,3 +637,47 @@ You can delete all your customer and delivery data by calling this command._
...
+| Logout   | `logout`                                                                                                                     | `logout`                                                                                                                    |
+| Delete   | `delete account`                                                                                                             | `delete account`                                                                                                            |
+
+#### Customer
+
+| Command | Format                                                                                   | Examples                                                                                          |
+|---------|------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| Add     | `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`           | `customer add --name Gabriel --phone 87654321 --email` |
```
Comment (B-enguin) [inline docs/UserGuide.md]: Is this missing an email

Comment (B-enguin) [reply docs/UserGuide.md]: Same as line 661

File: docs/UserGuide.md [hunk 8]
```diff
@@ -636,3 +637,49 @@ You can delete all your customer and delivery data by calling this command._
...
+| Recover  | `recover account [--answer ANSWER --password NEW_PASSWORD --confirmPass CONFIRM_PASSWORD]`                                   | `recover account --answer Koko --password newPassword123 --confirmPass newPassword123`                                      |
+| Logout   | `logout`                                                                                                                     | `logout`                                                                                                                    |
+| Delete   | `delete account`                                                                                                             | `delete account`                                                                                                            |
+
+#### Customer
+
+| Command | Format                                                                                   | Examples                                                                                          |
+| Add     | `customer add --name NAME --phone PHONE_NUMBER --email EMAIL --address ADDRESS`           | `customer add --name Gabriel --phone 87654321 --email` |
```
Comment (B-enguin) [inline docs/UserGuide.md]: Did add get duplicated here?

Comment (juliusgambe) [reply docs/UserGuide.md]: Shit I accidentally excluded it

**Review by B-enguin — APPROVED** [review]
LGTM

**Pull request merged**